packages/plans-grid/src/plans-grid/index.tsx

```diff
@@ -46,6 +50,10 @@ const PlansGrid: React.FunctionComponent< Props > = ( {
 	disabledPlans,
 	isExperimental,
 	locale,
+	showPlanTaglines = false,
+	CTAVariation = 'NORMAL',
+	popularBadgeVariation = 'ON_TOP',
```

Comment:
I don't think we need to maintain another variation of the PlansGrid in a non-accordion mode. However, it's non blocking for this PR. 

Just pinging @ollierozdarz to confirm so we can clean up later. Here is the old table version with badge on top and CTA non full-width.
<img width="159" alt="Screenshot 2020-11-18 at 13 37 36" src="https://user-images.githubusercontent.com/14192054/99526010-5062ea80-29a3-11eb-8a78-3fd5b9b0c7f4.png">